docs/src/.vuepress/config.js

```diff
@@ -155,6 +155,9 @@ module.exports = {
                 children: [{
                   text: 'Intro',
                   link: '/guides/smartweave/warp/readme.md'
+                }, {
+                  text: 'Deploying Contracts',
+                  link: '/guides/smartweave/warp/deploying-contracts.md'
```

Comment:
we need to decide on what naming convention we are using for files, im okay with kebab case, but others are using camel case or just all lower case.

Reply:
This can be cleaned up on a housing keeping issue, kebab is good for unix systems if we want to support caseSensitive and non-case system system., may not even be an issue, but consistency is important!